internal/cache/file/downloader/job.go

```diff
@@ -85,6 +85,8 @@ type Job struct {
 	removeJobCallback func()
 
 	mu locker.Locker
+
+	enableCrcCheck bool
```

Comment:
nit: add comment "Specifies whether Crc check needs to be done at the end of downloading the object into cache"